### DISCUSSION TITLE
Collection with developers

### DISCUSSION THREAD

**Discussion by phoenixmariepornstaractress** [eternnoir/pyTelegramBotAPI#2371] [General]
"Could I have your Telegram 🆔 Or Number so we can collaborate on some projects more effectively? I would greatly appreciate your help in expanding my code. Thank you."

**Comment by coder2020official** [comment]
Hi, we don't like discussing and collaborating on external projects.
However, if you have any questions, you are welcome to ask them in the chat:
https://t.me/joinchat/Bn4ixj84FIZVkwhk2jag6A
